src/js/components/Anchor/__tests__/Anchor-test.tsx

```diff
@@ -35,6 +35,11 @@ describe('Anchor', () => {
     expect(container).toMatchSnapshot();
   });
 
+  test('renders anchor without grommet wrapper', () => {
+    const { container } = render(<Anchor />);
+    expect(container).toMatchSnapshot();
```

Comment:
Good question, for a while we were using `asFragment` instead of `container` in our new snapshot tests based on a suggestion from a Kent C Dodds article on testing. The guidance has changed since then and the suggestion is to not use container if you are trying to query for elements in the dom. So we have been just using `asFragment` or `container.firstChild` for tests based on what the testing file is already using.

If you take a snapshot of the container your snapshot will include an extra div wrapper. If you take a snapshot of `container.firstChild` you won't get the div and will get the root element of your rendered element. I think generally it is a little cleaner to use `container.firstChild`

Additional info & links:

Container:
https://testing-library.com/docs/react-testing-library/api/#container-1

asFragment:
https://testing-library.com/docs/react-testing-library/api/#asfragment

Testing articles:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

https://kentcdodds.com/blog/effective-snapshot-testing